### PR TITLE
[release/2.1.5xx] Publish VS insertion nupkgs to blob feed

### PR DESCRIPTION
**Pull request by dagood**
Allows prodcon infrastructure to publish the VS insertion nupkgs.

Patterns copied from Publish.targets:

https://github.com/dotnet/cli/blob/b33df27d250772dadd4fda805ff65a3eb38a2b78/build/Publish.targets#L51-L54

### CODE REVIEW

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 1]
```diff
@@ -13,6 +13,20 @@
...
       </NupkgsForPublishing>
+
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.swr"
```
Comment (dagood) [inline build/publish/PublishNupkgToBlobFeed.targets]: Wait... this isn't a nupkg so I'm removing it, but I don't quite remember what it is.

Comment (nguerrera) [reply build/publish/PublishNupkgToBlobFeed.targets]: This file is needed along with the nupkg if the files change inside the nupkg. That is unlikely to ever happen in 2.x again, though. It is a manual process. As long as the swr is available somewhere, it's ok.

**Review by nguerrera — APPROVED** [review]

File: build/publish/PublishNupkgToBlobFeed.targets (hunk 1 above)
Comment (dagood) [reply build/publish/PublishNupkgToBlobFeed.targets]: Ok, cool. Yeah, it'll still be available in the blob feed as an asset, which makes it end up on the internal file share.

**Review by mmitche — APPROVED** [review]

**Comment by dsplaisted** [comment]
@nguerrera @mmitche Any reason not to go ahead and merge this (along with #13207)?

**Comment by mmitche** [comment]
> @nguerrera @mmitche Any reason not to go ahead and merge this (along with #13207)?

Go for it.

**Pull request merged**